exercises/hello-world/hello_world.py

```diff
@@ -1,2 +1,2 @@
-def hello():
+def hello(string):
```

Comment:
Judging by https://github.com/exercism/python/blob/master/exercises/hello-world/hello_world_test.py `hello-world` doesn't expect any parameters.
And `string` option is too general; `name` is much better

I suggest add a default parameter `def hello(name='')` to not confuse users